package.json

```diff
@@ -1,85 +1,85 @@
 {
```

Comment:
Can you kindly run an 
npm audit 
and 
npm audit fix 
This is to ensure that any known vulnerabilities are surfaced and fixed (where applicable).

Reply:
Hi @ramijebara I have ran the npm audit fix command. There was only one package, Please have a review once. Thank you.
![angular2gridster-npm-audit](https://user-images.githubusercontent.com/21095558/216884276-17b19d9e-36fc-4058-bccf-a94b3f115259.PNG)